### ISSUE TITLE
How to manage Joi.extend?

### ISSUE THREAD

**Issue by david-unergie** [Twipped/joi-to-swagger#48]
Hi,

we defined some joi extensions in our application, and we don't know if it's possible to manage that with j2s or not? Via `meta` or something else.
Currently, we get the error `TypeError: ourExtension is not a recognized Joi type`
```js
const joi = require('joi')
const joi2swagger = require('joi-to-swagger')
const ourJoi = joi.extend([
    {
        base: joi.string(),
        name: 'ourExtension',
        coerce(value, state, options) {
            return value + 'ourExtension'
        },
    },
])

joi2swagger({
    test: ourJoi.ourExtension()
})
// => TypeError: ourExtension is not a recognized Joi type
```

We would like to say to j2s `"ourExtension" is like "string"`, but we don't know how to do.
Is there a way?
Thanks

**Comment by david-unergie** [comment]
Hi @Twipped, do you have an idea about that? Do you think that is possible to manage joi extensions (how?) ? Or joi-to-swagger should be updated for that?

**Comment by Mairu** [comment]
Implemented with #57.
